pkpdapp/pkpdapp/views.py

```diff
@@ -20,3 +22,47 @@ class IndexView(TemplateView):
 
 class GenericView(TemplateView):
     template_name = 'generic.html'
+
+
+class PkpdModelDetailView(DetailView):
```

Comment:
Those would then go into the respective apps.

Reply:
Even if we don't split them up into different apps, we should probably have a `views` subdirectory that contains separate files for the different views. But for now there wasn't much in that one `view.py` file, so I left it combined

Reply:
Sounds good